tests/components/pi_hole/test_config_flow.py

```diff
@@ -37,17 +39,17 @@ async def test_flow_user(hass: HomeAssistant):
             user_input=CONFIG_FLOW_USER,
         )
         assert result["type"] == FlowResultType.FORM
-        assert result["step_id"] == "user"
-        assert result["errors"] == {CONF_API_KEY: "invalid_auth"}
+        assert result["step_id"] == "api_key"
+        assert result["errors"] == {}
```

Comment:
I'm missing a test for invalid api key error, ie `invalid_auth`.

Reply:
ah ... you are right ... just forgot to re-add it